src/assets/styles/variables.scss

```
@@ -0,0 +1,74 @@
+/* 디자인 변수 정의 */
+:root {
+  /* Font */
+  --font-family-base: 'Pretendard', sans-serif;
+
+  /* Font sizes */
+  --font-size-28: 28px;
+  --font-size-24: 24px;
+  --font-size-20: 20px;
+  --font-size-18: 18px;
+  --font-size-16: 16px;
+  --font-size-15: 15px;
+  --font-size-14: 14px;
+  --font-size-12: 12px;
+
+  /* Font weights */
+  --font-weight-regular: 400;
+  --font-weight-bold: 700;
+
+  /* Color - Purple */
+  --color-purple-100: #F8F0FF;
+  --color-purple-200: #ECD9FF;
+  --color-purple-300: #DCB9FF;
+  --color-purple-400: #C894FD;
+  --color-purple-500: #AB57FF;
+  --color-purple-600: #9935FF;
+  --color-purple-700: #861DEE;
+  --color-purple-800: #6E0AD1;
+  --color-purple-900: #5603A7;
+
+  /* Color - Beige */
+  --color-beige-100: #FFF0D6;
+  --color-beige-200: #FFE2AD;
+  --color-beige-300: #FFC583;
+  --color-beige-400: #FFAE65;
+  --color-beige-500: #FF8832;
+
+  /* Color - Blue */
+  --color-blue-100: #E2F5FF;
+  --color-blue-200: #B1E4FF;
+  --color-blue-300: #7CD2FF;
+  --color-blue-400: #34B9FF;
+  --color-blue-500: #00A2FE;
+
+  /* Color - Green */
+  --color-green-100: #E4FBDC;
+  --color-green-200: #D0F5C3;
+  --color-green-300: #9BE282;
+  --color-green-400: #60CF37;
+  --color-green-500: #2BA600;
+
+  /* Color - Grayscale */
+  --color-gray-100: #F6F6F6;
+  --color-gray-200: #EEEEEE;
+  --color-gray-300: #CCCCCC;
+  --color-gray-400: #999999;
+  --color-gray-500: #555555;
+  --color-gray-600: #4A4A4A;
+  --color-gray-700: #3A3A3A;
+  --color-gray-800: #2B2B2B;
+  --color-gray-900: #181818;
+
+  /* Color - White */
+  --color-white: #FFFFFF;
+
+  /* Color - Black */
+  --color-black:#000000;
```

```suggestion
  --color-black: #000000;
```
이 부분만 수정해주시면 될것 같아요 :)

---

넵 수정하였습니다!